app/components/task_list_items/validating/legislation_component.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  module Validating
```

Comment:
Should we be using `Validating` or `Validation`?

Reply:
The reason I did it `Validating` is because we already have `Reviewing`. By doing that, it will allow us to have a `validation` model in the future and not clash